frontend/src/components/eam/eam.tsx

```diff
@@ -66,7 +112,29 @@ const EAM: FC = () => {
       </h2>
       <Tenant />
       <div className={styles.tableWrapper}>
-        <WorkersTable>
+        <WorkersTable
+          footer={
+            <div className={styles.pagination}>
+              <div className={styles.countItems}>
+                <div className={styles.count}>{countItems + 'results'}</div>
+              </div>
+              <div className={styles.currentPage}>
+                <IconButton
+                  onClick={handleBackPage}
+                  icon={IconName.ARROW_LEFT}
+                  label="ArrowRight"
+                />
+                <div>{currentPage}</div>
+                <div className={styles.count}>of {allPage}</div>
+                <IconButton
+                  onClick={handleNextPage}
+                  icon={IconName.ARROW_RIGHT}
+                  label="ArrowRight"
+                />
+              </div>
+            </div>
+          }
```

Comment:
```suggestion
```
pagination should be in the common table, but props should be optional
and if we have all props that we need for pagination, we should render the pagination itself